DashKit/PodSDKProtocol/CannulaInsertion.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  CannulaInsertion.swift
+//  DashKit
+//
+//  Created by Pete Schwamb on 3/10/20.
+//  Copyright © 2020 Tidepool. All rights reserved.
+//
+
+import Foundation
+import PodSDK
+
+public protocol CannulaInsertion {
+    func insertCannula(eventListener: @escaping (ActivationStatus<ActivationStep2Event>) -> ())
+}
+
+extension DashPumpManager: CannulaInsertion {
+    public func insertCannula(eventListener: @escaping (ActivationStatus<ActivationStep2Event>) -> ()) {
+        let autoOffAlert = try! AutoOffAlert(enable: false)
```

Comment:
The throw from this initializer is to prevent invalid TimeIntervals from being passed in the `interval` parameter. We are not specifying `interval` and are using the default, so the throw will never happen.